zerver/views/registration.py

```diff
@@ -539,38 +539,33 @@ def find_account(request: HttpRequest) -> HttpResponse:
     url = reverse('zerver.views.registration.find_account')
 
     emails = []  # type: List[str]
+    emails_sent = []  # type: List[str]
```

Comment:
If you're gonna convert these to sets later, maybe just make them sets throughout the function?